src/Cookie/CookieWrapper.php

```diff
@@ -106,9 +103,22 @@ protected function set_cookie(
         $path,
         $domain,
         $secure,
+        $samesite,
         $httponly
     ): void {
-        \setcookie($name, $value, $expires, $path, $domain, $secure, $httponly);
```

Comment:
Ideally this method should be a bare proxy to the native call - so e.g. should just be:

```
protected function set_cookie($name, $value, $options) { setcookie($name, $value, $options);}
```

That way it's safe to stub it entirely in the test case, so you can see by looking that we're just capturing exactly what would otherwise go to the native method. All the logic about what the options are is in the calling method so the tests can assert against the final values.

So I guess the stub would then be:

```
protected function set_cookie($name, $value, $options) {
  $this->setcookie_calls[] = ['name' => $name, 'value' => $value, 'options' => $options];
}
```

And assertions updated to match.